src/dom-to-image-more.js

```diff
@@ -1010,45 +1012,147 @@
     let removeDefaultStylesTimeoutId = null;
     let tagNameDefaultStyles = {};
 
-    function getDefaultStyle(tagName) {
-        if (tagNameDefaultStyles[tagName]) {
-            return tagNameDefaultStyles[tagName];
+    const ascentStoppers = [
+        // these come from https://developer.mozilla.org/en-US/docs/Web/HTML/Block-level_elements
+        'ADDRESS',
+        'ARTICLE',
+        'ASIDE',
+        'BLOCKQUOTE',
+        'DETAILS',
+        'DIALOG',
+        'DD',
+        'DIV',
+        'DL',
+        'DT',
+        'FIELDSET',
+        'FIGCAPTION',
+        'FIGURE',
+        'FOOTER',
+        'FORM',
+        'H1',
+        'H2',
+        'H3',
+        'H4',
+        'H5',
+        'H6',
+        'HEADER',
+        'HGROUP',
+        'HR',
+        'LI',
+        'MAIN',
+        'NAV',
+        'OL',
+        'P',
+        'PRE',
+        'SECTION',
+        'SVG',
+        'TABLE',
+        'UL',
+        // these are ultimate stoppers in case something drastic changes in how the DOM works
+        'BODY',
+        'HEAD',
+        'HTML',
```

Comment:
[nit] The `svg` addition is ultimately custom - it's a performance boost!
SVG tags inside an `<svg>` are not going to have their default CSS change because of the parent elements outside of their `<svg>` tag:

```suggestion
        'TABLE',
        'UL',
        // these are ultimate stoppers in case something drastic changes in how the DOM works
        'BODY',
        'HEAD',
        'HTML',
        'SVG',
```

Reply:
You can also add `<math>` as the MathML sectioning root. It is not part of the HTML spec unfortunately, but web browsers irregularly implement it as a separate concurrent web technology

Reply:
`math` added, in lower case... I suspect when browsers actually do adopt the tag it'll morph to uppercase...

Reply:
Cool. We need to put `svg` (another lowercase exception) after the "these are ultimate stoppers" comment.  
Just to be sure it doesn't get deleted later when the list gets updated

Reply:
- We need to lowercase `svg` also!
- The `svg` tag won't appear in the MDN list so we should move it under the `// these are ultimate stoppers` comment

Reply:
Oh, doh... I can fix that real quick... completely missed what you were going for...